flask_common/utils.py

```diff
@@ -49,19 +47,24 @@ def json_list_generator(results):
             break
     yield ']'
 
-class isortedset(sortedset):
-    def __init__(self, *args, **kwargs):
-        if not kwargs.get('key'):
-            kwargs['key'] = lambda s: s.lower()
-        super(isortedset, self).__init__(*args, **kwargs)
-
-    def __contains__(self, key):
-        if not self:
-            return False
-        try:
-            return self[self.bisect_left(key)].lower() == key.lower()
-        except IndexError:
-            return False
+try:
+    from blist import sortedset
```

Comment:
so we end up with either `flask_common.utils.sortedset` or `flask_common.utils.isortedset` (notice the "i" prefix in the latter case) depending on the installed packages?